src/python/pants/testutil/pants_integration_test.py

```diff
@@ -199,7 +199,7 @@ def run_pants(
     command: Command,
     *,
     hermetic: bool = True,
-    use_pantsd: bool = True,
+    use_pantsd: bool = False,
```

Comment:
Can the argument just be dropped? If that doesn't break any callers today, it seems like it would be nice to steer folks wanting pantsd to run_pants_with_workdir and if they really want a throwaway one-off pantsd, they have to work a tiny bit harder.

Reply:
There certainly callers using `True`, however you likely can (and should) argue they aren't getting what they expect.

Reply:
Actually some cases aren't issuing multiple `run_pants` calls, but seem to just want to ensure the behavior is similar between the two modes. I don't feel confident enough that I would understand those tests enough to change them.

There certainly are some bad tests, but there also might be valid ones so a blanket change might not be a good fit.